src/test-run/commands/index.js

```diff
@@ -11,13 +18,34 @@ function selector (option, val) {
         throw new ActionSelectorTypeError(type);
 }
 
+function dragSourceSelector (option, val) {
+    var type = typeof val;
+
+    if (type !== 'string')
+        throw new DragSourceSelectorTypeError(type);
+}
+
+function dragDestinationSelector (option, val) {
+    var type = typeof val;
+
+    if (type !== 'string')
+        throw new DragDestinationSelectorTypeError(type);
+}
+
 function actionOptions (option, val) {
     var type = typeof val;
 
     if (type !== 'object' && val !== null && val !== void 0)
         throw new ActionOptionsTypeError(type);
 }
 
+function number (option, val) {
+    var valType = typeof val;
+
+    if (valType !== 'number' || isNaN(val))
+        throw new ActionNumberOptionError(option, valType);
+}
+
```

Comment:
Consider changes mad by @AndreyBelym in #452: fractional numbers check and `NaN` check. I guess we should better land #452 first.